README.md

```diff
@@ -200,6 +201,21 @@ You can use the Bluesky app to connect to your PDS.
 
 _Note: because the subdomain TLS certificate is created on-demand, it may take 10-30s for your handle to be accessible. If you aren't seeing your first post/profile, wait 30s and try to make another post._
 
+### Setting up SMTP
+
+To be able to verify users' email addresses and send other emails, you need to set up an SMTP server.
+
+One way to do this is to use an email service. [Resend](https://resend.com/) and [SendGrid](https://sendgrid.com/) are two popular choices.
+
+Create an account and API key on an email service and set these variables in `/pds/pds.env` (example with Resend):
```

Comment:
Here could add:

`Ensure that, if you are using a cloud provider, that they allow SMTP traffic by default. Even if your firewall rules open it, your cloud provider may have their own restrictions for spam`

Reply:
Thanks, let's see if the maintainers have an interest in merging this, then I can make the changes.